### PR TITLE
WINC-547: [wmco] Don't log when a service is not found

### PR DESCRIPTION
**Pull request by aravindhp**
Use the command "sc.exe qe" and the error code "1060" as hints to not log the error. We are doing this as it was spamming the log with no informational benefit.

### CODE REVIEW

**Comment by VaishnaviHire** [comment]
LGTM

File: pkg/controller/windowsmachine/windows/windows.go [hunk 1]
```diff
@@ -228,7 +234,10 @@ func (vm *windows) Run(cmd string, psCmd bool) (string, error) {
 
 	out, err := vm.interact.run(cmd)
 	if err != nil {
-		log.Error(err, "error running", "cmd", cmd, "out", out)
+		// Hack to not print the error log for "sc.exe qc" returning 1060 for non existent services.
+		if !(strings.HasPrefix(cmd, serviceQueryCmd) && strings.HasSuffix(err.Error(), serviceNotFound)) {
+			log.Error(err, "error running", "cmd", cmd, "out", out)
+		}
```
Comment (sebsoto) [inline pkg/controller/windowsmachine/windows/windows.go:240]: I'm worried about putting special cases like this in Run(), in the future we might have more cases like that. Did you consider not logging within `Run()` but logging at the callsite if the error is actually an error?

Comment (aravindhp) [reply pkg/controller/windowsmachine/windows/windows.go:240]: Yes I did. This means we have to keep ensuring folks log at the call site for new code. I prefer this for now and we can revisit if we keep adding exceptions.

**Comment by mansikulkarni96** [comment]
LGTM, realized we are now looking at the memory issues with tests in our CI as well.

File: pkg/controller/windowsmachine/windows/windows.go (hunk 1 above)
Comment (sebsoto) [reply pkg/controller/windowsmachine/windows/windows.go:240]: Sounds good to me

**Pull request merged**